compiler/vconone/src/version.test.cpp

```diff
@@ -45,5 +61,14 @@ TEST(vconone, copyright)
 {
   auto str = vconone::get_copyright();
 
+  int year = 2024;
+
+  if (year < get_current_year()){
```

Comment:
Should it fail the test? I am worried that production CI fails from this small problem.

Reply:
Or, what if, we make CI fail if it's not updated until Feburary of each year.

Reply:
Currently, printing warning is almost nothing because it's not printed unless `verbose` keyword is on for `./nncc test`